src/api/authorityApi.ts

```diff
@@ -78,7 +51,18 @@ export const createAuthority = async (user: User) => {
   try {
     const response = await Axios.post(url, { invertedname: `${user.familyName},${user.givenName}` }, { headers });
     if (response.status === StatusCode.OK) {
-      return response.data;
+      // TODO: check with backend why we suddenly get a list with one element as response when creating new authority
+      const systemControlNumber = response.data[0].systemControlNumber;
+      const updatedAuthority = await addQualifierIdForAuthority(
+        systemControlNumber,
+        AuthorityQualifiers.FEIDE_ID,
+        user.id
+      );
+      if (updatedAuthority) {
+        return updatedAuthority;
+      } else {
+        return response.data[0];
```

Comment:
hvis dette skjer så har vi fått opprettet authority, men ikke fått knyttet innlogget bruker til denne?
det betyr igjen at denne autoriteten vil dukke opp i listen for mulige autoriteter når man logger inn?
yes?

Reply:
Correct! Har laget en task på forespørsel om vi kan opprette autoritet ved å både sende invertedname og feideid, slik at vi kun gjør et kall. Men i mellomtiden må vi «manuelt» koble til